src/rebar_mix_builder.erl

```diff
@@ -30,4 +30,4 @@ format_error(Reason) ->
     io_lib:format("~p", Reason).
 
 sh(Command, Options) ->
-    rebar_utils:sh(Command, [{env, [{"ERL_FLAGS", ""}]} | Options]).
\ No newline at end of file
+    rebar_utils:sh(Command, Options).
```

Comment:
The call from `rebar_mix_hook:do/1` gets input `Options` "overridden" here. Nonetheless the `ERL_FLAGS` value is the same, so the end result is the same. This fix is out of semantics only.

I tried a couple of different approaches, all of which ended up being more convoluted then this simple change, which is innocuous (albeit more explicit) if the function is internal to the lib. only.